packages/metro-config/package.json

```diff
@@ -37,16 +37,16 @@
     "chalk": "^4.1.0",
     "debug": "^4.3.2",
     "getenv": "^1.0.0",
-    "metro-react-native-babel-transformer": "^0.59.0",
+    "metro-react-native-babel-transformer": "^0.64.0",
```

Comment:
```suggestion
```

And remove from metro-config, this was added in error. We should always read from the local project and throw if a package isn't available.